crypto/src/tests/ed25519_tests.rs

```diff
@@ -320,6 +320,29 @@ fn verify_batch_aggregate_signature_length_mismatch() {
     )
     .is_err());
 
+    // add an extra signature to both aggregated_signature that batch_verify takes in
```

Comment:
to mirror the sui test where sigs len is one extra longer than the len of pubkeys